docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+  val flippedChild = Flipped(Bool())
+}
+```
+
+Due to this, there are many desired connection behaviors between two Chisel components. First we will investigate a common source of confusion between port-direction and connection-direction. Then, we will dive in to the Chisel connection operators useful for connecting components with members of mixed-alignments.
+
+
+### Bi-direction connection operator (:<>=)
+
+For connections where you want 'bulk-connect-like-semantics' where the aligned members are driven producer-to-consumer and flipped members are driven consumer-to-producer, use `:<>=`.
+
+```scala mdoc:silent
+class Example1 extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  outgoing :<>= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are driven `incoming` to `outgoing` and flipped members are driven `outgoing` to `incoming`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example1())
+```
+
+### Port-Direction Computation versus Connection-Direction Computation
+
+A common question is if you use a mixed-alignment connection (such as `:<>=`) to connect submembers of parent components, does the alignment of the submember to their parent affect anything? The answer is no, because *alignment is always computed relative to what is being connected to, and members are always aligned with themselves.*
+
+In the following example connecting `incoming.alignedChild` to `outgoing.alignedChild`, whether `incoming.alignedChild` is aligned with `incoming` is irrelevant because the `:<>=` only computes alignment relative to the thing being connected to, and `incoming.alignedChild` is aligned with `incoming.alignedChild`.
+
+```scala mdoc:silent
+class Example1a extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  outgoing.alignedChild :<>= incoming.alignedChild // whether incoming.alignedChild is aligned/flipped to incoming is IRRELEVANT to what gets connected with :<>=
+}
+```
+
+While `incoming.flippedChild`'s alignment with `incoming` does not affect our operators, it does influence whether `incoming.flippedChild` is an output or input port of my module.
+A common source of confusion is to mistake the process for determining whether `incoming.flippedChild` is an output/input (the port-direction computation) with the process for determing how `:<>=` connects who to who (the connection-direction computation).
+While both processes consider relative alignment, they are distinct.
+
+The port-direction computation always computes alignment relative to the component marked with `IO`. An `IO(Flipped(gen))` is an input port, and any member of `gen` that is aligned/flipped with `gen` is an input/output port. An `IO(gen)` is an output port, and any member of `gen` that is aligned/flipped with `gen` is an output/input port.
```

Comment:
this isn't technically true, the port direction is actually something like `ActualDirection(BidirectionalDefault)`... do we care? What does DataMirror.directionOf return?